src/sign.builds

```diff
@@ -26,7 +26,7 @@
   </ItemDefinitionGroup>
```

Comment:
Instead of copying this condition in many places can you please create another property called "SkipFileSigning" or something along those lines that you default to true for arm64 and also allows us to set it from build definitions or the command line if we need to.